sample/src/main/java/org/walletconnect/example/wallet/WalletFragment.kt

```diff
@@ -0,0 +1,46 @@
+package org.walletconnect.example.wallet
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import org.walletconnect.example.R
+import org.walletconnect.example.databinding.WalletFragmentBinding
+import org.walletconnect.example.wallet.ui.SessionsAdapter
+import org.walletconnect.example.wallet.ui.UpdateActiveSessions
+import org.walletconnect.example.wallet.ui.sessionList
+
+class WalletFragment : Fragment(R.layout.wallet_fragment) {
+
+    private val viewModel: WalletViewModel by activityViewModels()
+    private lateinit var binding: WalletFragmentBinding
+    private val sessionAdapter = SessionsAdapter()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding = WalletFragmentBinding.bind(view)
+        viewModel.showBottomNav()
+        binding.sessions.apply {
+            layoutManager = LinearLayoutManager(context)
+            adapter = sessionAdapter
+        }
```

Comment:
With the update to the XML, this can be simplified to 
```suggestion
        binding.sessions.adapter = sessionAdapter
```